python/sglang/srt/models/deepseek_v2.py

```diff
@@ -1291,79 +1369,7 @@ def load_weights(self, weights: Iterable[Tuple[str, torch.Tensor]]):
                     )
                     weight_loader(param, loaded_weight)
 
-        if not global_server_args_dict["disable_mla"]:
```

Comment:
Good catch! Could you add fix to deepseek_nextn as well ?  https://github.com/sgl-project/sglang/blob/main/python/sglang/srt/models/deepseek_nextn.py#L265